File: analysis/ATP_rank_read.py
import os
import pandas as pd

folder_path = 'C:/Users/Paula/Documents/Projects/TennisData/ATP_ranking/rank_years/'
file_list = [os.path.join(folder_path, file) for file in os.listdir(folder_path) if file.endswith('.csv')]

all_dataframes = [pd.read_csv(file) for file in file_list]
df_ranks = pd.concat(all_dataframes, ignore_index=True)

File: analysis/ATP_rank_read.R
library(data.table)

folder_path <- "C:/Users/Paula/Documents/Projects/TennisData/ATP_ranking/rank_years/"
file_list <- list.files(path = folder_path, pattern = "*.csv", full.names = TRUE)
db <- rbindlist(lapply(file_list, fread), fill = TRUE)
